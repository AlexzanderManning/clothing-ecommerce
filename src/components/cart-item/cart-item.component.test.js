import { shallow } from "enzyme";
import "jest-enzyme";

import React from 'react';
import CartItem from './cart-item.component';

describe('Cart Item Component', () => {
  const mockItem = {
    item: {
    name: "Brown Brim",
    imageUrl: "https://i.ibb.co/ZYW3VTp/brown-brim.png",
    price: 25,
    quantity: 1
    }
  };

  const wrapper = shallow(<CartItem {...mockItem} />);

  it('renders to dom without crashing.', () => {
    expect.assertions(1);
    expect(wrapper).toMatchSnapshot();
  })
})
