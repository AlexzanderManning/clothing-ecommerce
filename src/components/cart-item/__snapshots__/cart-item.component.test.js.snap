// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cart Item Component renders to dom without crashing. 1`] = `
<div
  className="cart-item"
>
  <img
    alt="item"
    src="https://i.ibb.co/ZYW3VTp/brown-brim.png"
  />
  <div
    className="item-details"
  >
    <span
      className="name"
    >
      Brown Brim
    </span>
    <span
      className="price"
    >
      1
       x $
      25
    </span>
  </div>
</div>
`;
